controllers/suite_test.go

```diff
@@ -116,3 +120,42 @@ var _ = AfterSuite(func() {
 		Expect(testEnv.Stop()).To(Succeed())
 	}
 })
+
+func ContainRefOfGroupKind(group, kind string) types.GomegaMatcher {
+	return &refKindMatcher{
+		kind:  kind,
+		group: group,
+	}
+}
+
+type refKindMatcher struct {
```

Comment:
Please rename to something like refGroupKindMatcher

Reply:
Bump

Reply:
I did an incorrect rebase and lost that commit, adding it back